### ISSUE TITLE
some questions about how to run this project

### ISSUE THREAD

**Issue by XiaoBingCQY** [ziyuanzhangtony/GaitNet-CVPR2019#1]
Excuse me,the documents of this project are so many,so can you publish an more detailed instructions.Or which files are useless.Thank you very much.

**Comment by YunhaoDu** [comment]
Yes, I also hope you the author can give a more detailed README.md. 
Thank you!

**Comment by XiaoBingCQY** [comment]
Hello,I want to know whether you can run it?

**Comment by XiaoBingCQY** [comment]
@dyh-bupt Hello,I want to know whether you can run it?

**Comment by YunhaoDu** [comment]
@XiaoBingCQY  I'm sorry but I didn't try to run it.
